.goreleaser.yaml

```diff
@@ -1,5 +1,3 @@
-partial:
-  by: target
```

Comment:
Why was this removed?

Reply:
Running locally, I saw a warning that goreleaser files were out of sync, so I ran `make goreleaser`. Given that I got an issue about the runner running out of space, I reverted this change for now.